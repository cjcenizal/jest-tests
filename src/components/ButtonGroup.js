import React from 'react';
import { Button } from './Button';

export const ButtonGroup = () => {
  return [
    <Button id="save" key="save">
      <span className="fa fa-save"/><span>Save</span>
    </Button>,

    <Button id="delete" key="delete">
      <span className="fa fa-trash"/><span>Delete</span>
    </Button>,

    <Button id="cancel" key="cancel">
      <span className="fa fa-undo"/><span>Cancel</span>
    </Button>,

    <Button id="copy" key="copy">
      <span className="fa fa-copy"/><span>Copy</span>
    </Button>,

    <Button id="edit" key="edit">
      <span className="fa fa-pencil"/><span>Edit</span>
    </Button>
  ];
};
