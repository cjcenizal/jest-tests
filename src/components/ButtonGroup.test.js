import React from 'react';
import { shallow } from 'enzyme';
import { ButtonGroup } from './ButtonGroup';

describe('ButtonGroup', () => {
  test('is rendered', () => {
    const component = shallow(<ButtonGroup />);

    expect(component).toMatchSnapshot();
  });
});
