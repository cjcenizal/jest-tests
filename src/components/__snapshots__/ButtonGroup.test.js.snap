// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup is rendered 1`] = `
Array [
  <Button
    id="save"
    key="save"
  />,
  <Button
    id="delete"
    key="delete"
  />,
  <Button
    id="cancel"
    key="cancel"
  />,
  <Button
    id="copy"
    key="copy"
  />,
  <Button
    id="edit"
    key="edit"
  />,
]
`;
